app/controllers/OrganisationDetailsController.scala

```diff
@@ -29,12 +29,41 @@ import scala.concurrent.Future
 trait OrganisationDetailsController extends LisaBaseController {
   val rosmService:RosmService
 
+  def businessLables(businessStructure:  Option[BusinessStructure]): String = {
```

Comment:
As this is in the controller I'd suggest making it a private def. Makes little to no difference to the application as-is but it'll demarcate it further from the methods which handle requests & should increase readability.